packages/jest-reporters/src/coverage_reporter.ts

```diff
@@ -405,8 +432,84 @@ export default class CoverageReporter extends BaseReporter {
     }
   }
 
-  // Only exposed for the internal runner. Should not be used
-  getCoverageMap(): istanbulCoverage.CoverageMap {
-    return this._coverageMap;
+  private async _getCoverageResult(): Promise<{
+    map: istanbulCoverage.CoverageMap;
+    reportContext: istanbulReport.Context;
+  }> {
+    if (this._globalConfig.v8Coverage) {
+      const mergedCoverages = mergeProcessCovs(
+        this._v8CoverageResults.map(cov => ({result: cov.map(r => r.result)})),
+      );
+
+      const fileTransforms = new Map<string, TransformResult>();
+
+      this._v8CoverageResults.forEach(res =>
+        res.forEach(r => {
+          if (r.codeTransformResult && !fileTransforms.has(r.result.url)) {
+            fileTransforms.set(r.result.url, r.codeTransformResult);
+          }
+        }),
+      );
+
+      const transformedCoverage = await Promise.all(
+        mergedCoverages.result.map(async res => {
+          const fileTransform = fileTransforms.get(res.url);
+
+          let sourcemapContent: FixedRawSourceMap | undefined = undefined;
+
+          if (
+            fileTransform &&
+            fileTransform.sourceMapPath &&
+            fs.existsSync(fileTransform.sourceMapPath)
+          ) {
+            sourcemapContent = JSON.parse(
+              fs.readFileSync(fileTransform.sourceMapPath, 'utf8'),
+            );
+          }
+
+          const converter = v8toIstanbul(
+            res.url,
+            0,
+            fileTransform && sourcemapContent
+              ? {
+                  originalSource: fileTransform.originalCode,
+                  source: fileTransform.code,
+                  sourceMap: {sourcemap: sourcemapContent},
+                }
+              : {source: fs.readFileSync(res.url, 'utf8')},
+          );
+
+          await converter.load();
+
+          converter.applyCoverage(res.functions);
+
+          return converter.toIstanbul();
+        }),
+      );
+
+      const map = istanbulCoverage.createCoverageMap({});
+
+      transformedCoverage.forEach(res => map.merge(res));
+
+      const reportContext = istanbulReport.createContext({
+        // @ts-ignore
```

Comment:
What's up with all the ts-ignores in here?

Reply:
the istanbul types has not been updated for the new major